server/routes/updateGoal/reviewView.ts

```diff
@@ -0,0 +1,16 @@
+import type { PrisonerSummary } from 'viewModels'
+import type { UpdateGoalForm } from 'forms'
+
+export default class ReviewView {
```

Comment:
I'm thinking the name `ReviewView` is a bit vague / nondescript (even though it is in a folder structure which arguably gives it some context) ... 🤔 

What about something like `reviewUpdateGoalView.ts` where the classname is `ReviewUpdateGoalView` ?

What do you think? 🤔